server/tests/src/test/java/org/infinispan/server/persistence/JdbcConfigurationUtil.java

```diff
@@ -0,0 +1,63 @@
+package org.infinispan.server.persistence;
+
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.persistence.jdbc.configuration.JdbcStringBasedStoreConfigurationBuilder;
+import org.infinispan.persistence.jdbc.configuration.PooledConnectionFactoryConfigurationBuilder;
+import org.infinispan.server.test.core.persistence.Database;
+import org.infinispan.util.concurrent.IsolationLevel;
+
+public class JdbcConfigurationUtil {
+
+    private PooledConnectionFactoryConfigurationBuilder persistenceConfiguration;
+    private ConfigurationBuilder configurationBuilder;
+    private CacheMode cacheMode;
+
+    public JdbcConfigurationUtil(CacheMode cacheMode) {
+        configurationBuilder = new ConfigurationBuilder();
+        this.cacheMode = cacheMode;
+    }
+
+    public JdbcConfigurationUtil createPersistenceConfiguration(Database database, boolean passivation, boolean preload) {
```

Comment:
This method is always called after directly after the instance is created, e.g. `new JdbcConfigurationUtil().createPersistenceConfiguration()` ... why not just add these params to the constructor so that we can just do:

`new JdbcConfigurationUtil(CacheMode.REPL_SYNC, database, false, false).setEviction()` etc.